### DISCUSSION TITLE
Problem about addImputeWeights

### DISCUSSION THREAD

**Discussion by hahahah-single** [GreenleafLab/ArchR#1374] [Questions / Documentation]
When I use creatArrowFiles function I meet the warning :Warning message :
in sprintf("Completed Getting Magic Weights!", round(object.size(weightList)/10^9, :“one argument not used by format 'Completed Getting Magic Weights!'” 

I would like to ask what is the reason for this and can i ignore this?

**Comment by rcorces** [comment]
I have never encountered this before and I'm not sure why this would be called during Arrow file creation. Certainly, this would indicate that the `addImputeWeights()` function is not working correctly.

**Reply by hahahah-single** [reply]
Very sorry for my wrong description! I meet the warning when I use addImputeWeights function！But when I use
p <- plotEmbedding(ArchRProj = projHeme2, colorBy = "GeneScoreMatrix", name = markerGenes, embedding = "UMAP",imputeWeights = getImputeWeights(projHeme2)) .it runs successfully,So is there any way I can fix this？

**Reply by rcorces** [reply]
what happens when you just run `getImputeWeights(projHeme2)`?

**Reply by hahahah-single** [reply]
it just show ：
Getting ImputeWeights

**Reply by rcorces** [reply]
please upload your logfile when you run the `addImputeWeights()` functino

**Reply by hahahah-single** [reply]
At the end of this document it return Completed Getting Magic Weights!, 1.089 mins elapsed.
I found that it might be because of R language when I run sprintf("Completed Getting Magic Weights!", 3) it also return the same warning :one argument not used by format.

**Reply by rcorces** [reply]
Sorry for not realizing this sooner. You're correct - this is a mistake in the code because the sprintf statement isnt complete. This should be fixed now via https://github.com/GreenleafLab/ArchR/commit/3075f1f034d8c5a9d7dcdfd6b6990814733fbf87 on the `release_1.0.2` branch.

**Reply by hahahah-single** [reply]
Thank you very much, I would like to ask if this will affect the addImputeWeights output result? Can I ignore this warning and continue the analysis？

**Reply by rcorces** [reply]
yes you can ignore this warning. it will not affect the results.
